alcs-api/src/decision/application-amendment/application-amendment.entity.ts

```diff
@@ -28,16 +28,16 @@ export class ApplicationAmendment extends Base {
   submittedDate: Date;
 
   @AutoMap()
-  @Column({ nullable: true })
-  isReviewApproved: boolean;
+  @Column({ type: 'boolean', nullable: true })
+  isReviewApproved: boolean | null;
 
   @AutoMap()
   @Column()
   isTimeExtension: boolean;
 
   @AutoMap()
   @Column({ type: 'timestamptz', nullable: true })
-  reviewDate: Date;
+  reviewDate: Date | null;
```

Comment:
Is it always supplied? Do we need to include undefined so it the property will be left unchanged if it is not provided?